src/azul/deployment.py

```diff
@@ -347,13 +346,17 @@ def _populate_tags(tf_config: JSON) -> JSON:
 
 
 def emit_tf(tf_config: Optional[JSON]):
-    return emit(tf_config) if tf_config is None else emit(_sanitize_tf(_populate_tags(tf_config)))
+    return emit(tf_config) if tf_config is None else emit(_sanitize_tf(populate_tags(tf_config)))
 
 
-def tags(resource_name: str, **kwargs) -> Dict[str, str]:
+def tags(resource_name: str, **kwargs) -> JSON:
```

Comment:
I sort of think the previous annotation was more accurate, right? AWS won't accept full JSON for a tag, I think, only a key-value pair.

Reply:
This is something that I went back and forth with for a while. After thinking about it more, I think you're right and I'll change it back.

My rationale was that the `JSON` annotation was accurate for consistency with `def _populate_tags(tf_config: JSON) -> JSON` which itself is consistent with `def _sanitize_tf(tf_config: JSON) -> JSON`. That being said, the linter doesn't like it, because I rely on some `dict` methods through the code. The other option would be `MutableJSON` (alias of `Dict[str, AnyMutableJSON]`), but that doesn't seem to be appropriate here.